### PR TITLE
Add functions to get user timeout limits

### LINKED ISSUE #15: [limits] Support user timeout limits
Add method to retrieve user timeout limits

### PR DESCRIPTION
**Pull request by dgaubert**
Fixes #15

### CODE REVIEW

**Comment by rochoa** [comment]
Should we deprecate the `limits:tiler:vizzuality render 2000` case? Or is that still relevant?

File: lib/carto_metadata.js [hunk 1]
```diff
@@ -499,6 +500,30 @@ module.exports = function(redis_opts) {
...
+
+            for (var i = 0, len = limitsParamsKeys.length; i < len; i++) {
+                paramsMap[limitsParams[limitsParamsKeys[i]]] = (!!limitsValues[i]) ? +limitsValues[i] : limitsValues[i];
+            }
+
+            callback(null, paramsMap);
+        });
+    };
+
+    me.getUserTimeoutRenderLimits = function(username, authorized, callback) {
```
Comment (rochoa) [inline lib/carto_metadata.js]: Isn't a bit weird to have an authorized param at this layer?

**Comment by dgaubert** [comment]
I've tried to keep it. If present we apply it instead of user render timeout. There are a few users with that configuration in prod, if I recall correctly they are outliers.

**Comment by rochoa** [comment]
Then, I would remove the `limits:tiler:$user` approach. There are only a few users, most likely, as you said, due to outliers. I prefer to keep it simple and have fewer alternatives to do the same.

**Comment by dgaubert** [comment]
It's ready for another CR round!

File: test/support/prepare_db.sh [hunk 2]
```diff
@@ -27,6 +27,7 @@ HMSET rails:users:vizzuality id 1 \
                              database_password secret map_key 1234
 SADD rails:users:vizzuality:map_key 1235
 HSET limits:tiler:vizzuality render 2000
```
Comment (rochoa) [inline test/support/prepare_db.sh]: I would remove this.

Comment (dgaubert) [inline test/support/prepare_db.sh]: F**k, I forgot it.

Comment (dgaubert) [inline test/support/prepare_db.sh]: Done!

**Comment by dgaubert** [comment]
@rochoa, is ti ready to merge?

**Comment by dgaubert** [comment]
@rochoa, if you don't mind I'm going to proceed with this. All required changes were implemented. 

You have 5 min to stop me. 😈

**Comment by dgaubert** [comment]
3, 2, 1 ... meging

**Pull request merged**